nixos/doc/manual/configuration/file-systems.xml

```diff
@@ -16,6 +16,12 @@
     fsType = "ext4";
   };
 </programlisting>
+  This will create the <filename>fstab</filename> entry and a
+  <command>systemd</command> unit file for the filesystem, as well as
+  automatically mount it, unless <literal>"noauto"</literal> is present in
+  <link linkend="opt-fileSystems._name__.options">options</link>.
+  <literal>"noauto"</literal> filesystems can be mounted explicitly using
+  <command>systemctl</command> e.g. <command>systemctl start data.mount</command>.
```

Comment:
```suggestion
  This will create an entry in <filename>/etc/fstab</filename>, which will generate
  a corresponding <link xlink:href="https://www.freedesktop.org/software/systemd/man/systemd.mount.html">systemd.mount</literal> unit via <link xlink:href="https://www.freedesktop.org/software/systemd/man/systemd-fstab-generator.html">systemd-fstab-generator</link>.
  As long as <literal>"noauto"</literal> isn't present in <link linkend="opt-fileSystems._name__.options">options</link>, it will be mounted automatically.
   <literal>"noauto"</literal> filesystems can be mounted explicitly using
  <command>systemctl</command> e.g. <command>systemctl start data.mount</command>.
```

(written in the web interface, please reflow)

Reply:
I found the wording of `as long as x is not present` a little clumsy so I switched it back to `unless x is present`, what do you think?

Reply:
Also, is there a way to reflow long URLs like the ones in your edit?

Reply:
IIRC, if you `nix-shell` in the doc folder, there should be a makefile that should lint things.

Reply:
I tried that and it reflowed everything in every file to a single long line. You're talking about e.g. https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/Makefile#L13, right?

Reply:
Hm, then it's probably as it is :-)